src/plugins/console/public/application/containers/split_panel/panel.tsx

```diff
@@ -7,7 +7,7 @@
  */
 
 import React, { CSSProperties, ReactNode, useEffect, useRef, useState } from 'react';
-import { usePanelContext } from '../context';
+import { usePanelContext } from '../../contexts/split_panel_context';
```

Comment:
Please, add the export of `usePanelContext` to the `contexts/index.ts` and change corresponding imports.